File: frontend/src/app/fizioterapeut/fizioterapeutDodajRequest.ts
export interface FizioterapeutDodajRequest
{
  imePrezime: string
  jmbg: string
  datumRodjenja: string
  datumZaposlenja: string
  nalogId: number | null
  poslovnaPozicijaId: number
  oblastFizijatrije: string
}

File: DomZaStaraLicaApi/Endpoints/Fizioterapeut/Dodaj/FizioterapeutDodajRequest.cs
namespace DomZaStaraLicaApi.Endpoints.Fizioterapeut.Dodaj
{
    public class FizioterapeutDodajRequest
    {
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int? NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public string OblastFizijatrije { get; set; }
    }
}
